otc-watcher/README.md

```diff
@@ -0,0 +1,82 @@
+# otc-watcher
+
+Watches addresses on blockchain and stores output information.
+
+*NOTE*: Currently only BTC address watching is supported. Adding support for other currencies in the future will be easy, thanks to the [currency connection interface](pkg/currency/currency.go).
+
+## running
+
+Right now the only dependency is to have `btcwallet` running. Then you can start `otc-watcher` with the following command (assuming you ran `go build`):
+
+```
+./otc-watcher -rpc_node="localhost:8332" \
+              -rpc_user="username"       \
+              -rpc_pass="password"       \
+              -wallet_account="account"  \
+              -wallet_pass="password"    \
```

Comment:
Maybe you should rename it to addr or listenstr, for the case when you want to specify to listen only localhost.